src/splat/segtypes/n64/img.py

```diff
@@ -76,12 +75,11 @@ def should_split(self) -> bool:
         return options.opts.is_mode_active("img")
 
     def split(self, rom_bytes):
+        self.check_len()
+
         path = self.out_path()
         path.parent.mkdir(parents=True, exist_ok=True)
 
-        assert isinstance(self.rom_start, int)
-        assert isinstance(self.rom_end, int)
-
```

Comment:
Aren't those asserts kinda important?

Reply:
they're in check_len() anyway, which runs before